### PR TITLE
Refactored and applied comments

### PR DESCRIPTION
**Pull request by chriskn**
I am not shure about the requirements regarding the bridge id. We should clarify that with Philiphs

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/BridgeJsonParameters.java [hunk 1]
```diff
@@ -7,39 +7,44 @@
...
 
+import com.google.gson.annotations.SerializedName;
+
 /**
  * The {@link BridgeJsonParameters} class defines JSON object, which
  * contains bridge attributes like IP address. It is used for bridge
  * N-UPNP Discovery.
  *
- * @author Awelkiyar Wehabrebi
+ * @author Awelkiyar Wehabrebi - Initial contribution and API
```
Comment (afuechsel) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/BridgeJsonParameters.java:17]: @chriskn you need to add yourself as author too, I gues

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/BridgeJsonParameters.java [hunk 2]
```diff
@@ -7,39 +7,44 @@
...
-    // hue bridge parameters
+    @SerializedName("id")
     private final String id;
-    private final String internalipaddress;
-    private final String macaddress;
+    @SerializedName("internalipaddress")
+    private final String internalIpAddress;
+    @SerializedName("macaddress")
+    private final String macAddress;
+    @SerializedName("name")
```
Comment (afuechsel) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/BridgeJsonParameters.java]: that's not needed

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/BridgeJsonParameters.java [hunk 3]
```diff
@@ -7,39 +7,44 @@
...
     private final String name;
 
     public BridgeJsonParameters(String id, String internalipaddress, String macaddress, String name) {
```
Comment (afuechsel) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/BridgeJsonParameters.java]: I would also camel case the parameters

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java [hunk 4]
```diff
@@ -36,23 +40,35 @@
  */
 public class HueBridgeNupnpDiscovery extends AbstractDiscoveryService {
 
-    private final Logger logger = LoggerFactory.getLogger(HueBridgeNupnpDiscovery.class);
+    private static final String MODEL_NAME_PHILIPS_HUE = "<modelName>Philips hue";
+
+    private static final String BRIDGE_INDICATOR = "fffe";
```
Comment (afuechsel) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java:45]: this been clarified with Philips?

**Review by afuechsel — CHANGES_REQUESTED** [review]
still no test. don't we need one?

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java (hunk 4 above)
Comment (chriskn) [reply extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java:45]: Not yet. I am waiting for news from Simon

**Pull request merged**